### PR TITLE
Lab 7 Completion

### PR DESCRIPTION
**Pull request by compugal99**
We have completed Lab 7 with all of its stuff and documented results

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Alex and Henry Lab 7
+
```
Comment (TechnicallyRy) [inline README.md:2]: Nice descriptions on the overview of each activity section.

**Review by TechnicallyRy — APPROVED** [review]
Great separation of concerns for each activity section, your structure with the if def fields works great with the CAN modes. Nice job!

File: src/main.c [hunk 2]
```diff
@@ -1,5 +1,83 @@
...
+#include <device.h>
+#include <devicetree.h>
+#include <can/can_loopback.h>
+#include "periodMsg.h"
+#include "receive_callback.h"
+#include "priority_msg.h"
+
+typedef struct k_thread k_thread_t;
+
+#define STACKSIZE 2000
```
Comment (Dima5877) [inline src/main.c:11]: Adequate stack size leaving room for debugging.

**Review by Dima5877 — COMMENTED** [review]
Concise and efficient.

File: README.md (hunk 1 above)
Comment (Dima5877) [reply README.md:2]: I concur.

**Review by Dima5877 — APPROVED** [review]
Good work.

**Pull request merged**